src/com/codeenginestudio/bookManagement/servlets/EditBookServlet.java

```diff
@@ -0,0 +1,64 @@
+package com.codeenginestudio.bookManagement.servlets;
+
+import java.io.IOException;
+
+import javax.servlet.RequestDispatcher;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import com.codeenginestudio.bookManagement.manage.ManageBook;
+import com.codeenginestudio.bookManagement.models.Book;
+
+/**
+ * Servlet implementation class EditBookServlet
+ */
+@WebServlet("/EditBookServlet")
+public class EditBookServlet extends HttpServlet {
+
+    public void displayView(HttpServletRequest request, HttpServletResponse response, String fileName)
```

Comment:
This method is using many times. You should move it to a new utilize class or somethings like that.